public/main.c74709ea561d3a8b1ace.hot-update.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"main.c74709ea561d3a8b1ace.hot-update.js","mappings":";;;;;;;;;;;;;;AAAA,wEAA8B;AAE9B,mGAA0B;AAC1B,8FAAmC;AACnC,mHAAgD;AAEhD,gIAAkD;AAElD,gFAAgF;AAEhF,QAAQ,CAAC,gBAAgB,CAAC,kBAAkB,EAAE,GAAG,EAAE,CACjD,kBAAM,CACJ,8BAAC,+BAAY;IACX,8BAAC,cAAI,OAAG,CACK,EACf,QAAQ,CAAC,cAAc,CAAC,MAAM,CAAC,CAChC,CACF,CAAC;;;;;;;;;UCjBF","sources":["webpack://chime-sdk-react-demo/./src/apps/attendee/index.tsx","webpack://chime-sdk-react-demo/webpack/runtime/getFullHash"],"sourcesContent":["import '../../app.global.css';\n\nimport React from 'react';\nimport { render } from 'react-dom';\nimport { AppContainer } from 'react-hot-loader';\n\nimport Root from '../../components/attendee/Root';\n\n// const AppContainer = process.env.PLAIN_HMR ? Fragment : ReactHotAppContainer;\n\ndocument.addEventListener('DOMContentLoaded', () =>\n  render(\n    <AppContainer>\n      <Root />\n    </AppContainer>,\n    document.getElementById('root')\n  )\n);\n","__webpack_require__.h = () => (\"d31dfb952066e8ea4f92\")"],"names":[],"sourceRoot":""}
```

Comment:
What are these files for? Do we need to commit them?

Reply:
removed these. Public folder was accidentally commited. I added it to the gitignore.